classes/locations.php

```diff
@@ -201,11 +201,11 @@ function premium_support() {
 	else :
 
 	?>
-	<p class='howto aligncenter'><?php printf( __( 'Your premium support license for %s is valid until %s', 'simplemap' ), get_ftps_site( $simplemap_ps ), date( 'F d, Y', get_ftps_exp_date( $simplemap_ps ) ) ); ?></p>
+	<p class='howto aligncenter'><?php printf( __( 'Your premium support license for %s is valid until %s', 'SimpleMap' ), get_ftps_site( $simplemap_ps ), date( 'F d, Y', get_ftps_exp_date( $simplemap_ps ) ) ); ?></p>
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 217 characters

<!-- Comment by [SideCI](https://sideci.com) -->